tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -60,6 +60,10 @@ def _run(self):
                     SyncObject(data=json.dumps(raw_data))
                 sync_object.save()
 
+                gluster_brick_dir = NS.gluster.objects.\
+                                     GlusterBrickDir()
+                gluster_brick_dir.save()
```

Comment:
We should have separation between tendrl internal config vs user config, ill file a bug for this on specs after next build and figure out how much work is needed.

For now, this seems fine. @nnDarshan you can create and save GlusterBrickDir in the first sync only and skip it for any other syncs